### PR TITLE
aqlm

### PR DESCRIPTION
**Pull request by flozi00**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Was this discussed/approved via a Github issue or the discord / slack channel? Please add a link
      to it if that's the case.
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by flozi00** [comment]
@tgaddair any idea why to docker copy statement fails ?

**Comment by flozi00** [comment]
@tgaddair 
I dont know whats going on with me today
cloud you please finish this PR ?
Need to take a break for some days

**Comment by thincal** [comment]
+1 for the AQLM requirement

File: server/lorax_server/models/__init__.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def get_model(
                 trust_remote_code=trust_remote_code,
             )
 
-    if model_type == "llama":
+    if model_type in ["llama", "llama_aqlm"]:
```
Comment (BlackSamorez) [inline server/lorax_server/models/__init__.py:237]: `llama_aqlm` and similar custom model types are deprecated. We moved AQLM from custom code to transformers integration with `transformers` `4.38.0`. Correct me if I'm wrong and it serves different purpose here.

File: server/pyproject.toml [hunk 2]
```diff
@@ -38,13 +38,14 @@ boto3 = "^1.28.34"
 urllib3 = "<=1.26.18"
 hqq = { version = "^0.1.2", optional = true }
 stanford-stk = { version = "^0.7.0", markers = "sys_platform == 'linux'" }
+aqlm = { version = "^1.0.0"}
```
Comment (BlackSamorez) [inline server/pyproject.toml:41]: Proper backprop was added in `aqlm==1.1.0` and bf16 support was added in `aqlm==1.1.2`.

**Pull request closed without merging**